src/main.js

```diff
@@ -777,10 +787,19 @@ class MiniGraphCard extends LitElement {
     }
     const dec = this.config.decimals;
     if (dec === undefined || Number.isNaN(dec) || Number.isNaN(state))
-      return Math.round(state * 100) / 100;
+      return this.formatStateValue(Math.round(state * 100) / 100, 0);
 
     const x = 10 ** dec;
-    return (Math.round(state * x) / x).toFixed(dec);
+    return this.formatStateValue((Math.round(state * x) / x).toFixed(dec), dec);
+  }
+
+  formatStateValue(v, dec) {
+    const _locales = this.config.locales;
+    if (_locales === null || _locales === undefined) {
+      return v;
+    } else {
+      return parseFloat(v, 10).toLocaleString(_locales, { maximumFractionDigits: Number(dec) });
+    }
   }
```

Comment:
Could replace with
```js
formatStateValue(state, maximumFractionDigits) {
  if (this.config.locale) {
    return parseFloat(state, 10).toLocaleString(this.config.locale, { maximumFractionDigits });
  }
  return state;
}
```

Not sure if we need the parseFloat() since we always do Number() or parseFloat() in computeState() before calling this function.